src/components/Routes.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
Shouldn't this be `import * as React from 'react';`?

Reply:
Check

Reply:
:+1: